src/ng-daterangepicker/ng-daterangepicker.component.ts

```diff
@@ -89,15 +90,11 @@ export class NgDateRangePickerComponent implements ControlValueAccessor, OnInit,
   ngOnInit() {
     this.opened = false;
     this.date = dateFns.startOfDay(new Date());
-    this.options = this.options || this.defaultOptions;
+    this.options = _.extend(this.options || {}, this.defaultOptions);
```

Comment:
You can but I need the same pointer and I don't want defined properties to be overwritten.

`Properties in the target object will be overwritten by properties in the sources if they have the same key.  Later sources' properties will similarly overwrite earlier ones.  `
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign